packages/contracts/contracts/SmartEscrowMono.sol

```diff
@@ -218,7 +218,7 @@ contract SmartEscrowMono is Context, IArbitrable, ReentrancyGuard {
   function payDisputeFees(address _adr) internal {
     IArbitrator arbitrator = IArbitrator(_adr);
     (, IERC20 feeToken, uint256 feeAmount) = arbitrator.getDisputeFees();
-    feeToken.safeTransferFrom(msg.sender, address(this), feeAmount);
+    feeToken.safeTransferFrom(msg.sender, address(this), feeAmount); // sender must pay dispute fees when locking
```

Comment:
> line 206: how are we determining how to pay the dispute fees?

as commented, in the UI we will request the user who's requesting dispute resolution i.e., calling the `lock` method to pay the fees.
We must `approve` the token for that much amount before calling this function.

Reply:
Could we add a check to see whether the dispute token matches the locker token? If so, we could pay from the locker similar to how the lexdao resolution fee works.